index.d.ts

```diff
@@ -1532,13 +1532,13 @@ export interface Schema<A extends string, F extends string, C extends string> {
             readonly pk: {
                 readonly casing?: "upper" | "lower" | "none" | "default";
                 readonly field: string;
-                readonly composite: ReadonlyArray<F>;
+                readonly composite?: ReadonlyArray<F>;
```

Comment:
Hey @Max101 👋

While this is supported by the library, the type-system depends on this composite being present which is why there is a little disconnect here between the types and the validation.

Reply:
right ok I understand. We have defaulted to an empty array for now